internal/vrf/ips.go

```diff
@@ -0,0 +1,65 @@
+package vrf
+
+import (
+	"context"
+	"fmt"
+
+	metal "github.com/equinix/equinix-sdk-go/services/metalv1"
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) ips() *cobra.Command {
+	var (
+		vrfID string
+		ipID  string
+	)
+
+	// createVRFCmd represents the creatVRF command
+	ipsVRFCmd := &cobra.Command{
+		Use:   "ips [-i <VrfID] [-p <ip_id>]",
```

Comment:
The help text says `-i` but the argument definition says `-v`. 

I think we'll want:

`ips (-v <vrf_id> | -i <ip_id>)`

Reply:
Resolving this one as well because it became disconnected from the relevant code.  There is a similar unaddressed review comment on the latest code.